src/worker/requirements.txt

```diff
@@ -17,9 +17,10 @@ environs==9.5.0
 exceptiongroup==1.1.2
 frozenlist==1.4.0
 googleapis-common-protos==1.60.0
+greenlet==3.0.0
 grpc-gateway-protoc-gen-openapiv2==0.1.0
-grpcio
-grpcio-tools
+grpcio==1.56.0
```

Comment:
You need to manually delete `==1.56.0` because it will break stuff in the docker build. you have to let pip resolve the dependency by itself

Reply:
Made the requested change